pkgs/by-name/co/cosmic-settings/package.nix

```diff
@@ -36,6 +37,14 @@ rustPlatform.buildRustPackage (finalAttrs: {
     hash = "sha256-UKg3TIpyaqtynk6wLFFPpv69F74hmqfMVPra2+iFbvE=";
   };
 
+  patches = [
+    # TODO: This is merged and will be included in the 7th Alpha release, remove it then.
+    (fetchpatch {
+      url = "https://github.com/pop-os/cosmic-settings/commit/2e2898b31fabcb3f3b561e6c4cea2aca9de9b284.patch";
```

Comment:
This patch doesn't look up wallpapers in `XDG_DATA_DIR`, instead it tries to guess the operating system and uses one of two hardcoded paths.  It'd be nicer if it _did_ use `XDG_DATA_DIR`, but if it's not going to, the commit message should at least be corrected.

Reply:
Opened https://github.com/pop-os/cosmic-settings/issues/1120